dmoj/commands/test.py

```diff
@@ -0,0 +1,122 @@
+import os
+import sys
+import traceback
+from operator import itemgetter
+
+import yaml
+
+from dmoj import executors
+from dmoj.commands.base_command import Command
+from dmoj.error import InvalidCommandException
+from dmoj.judgeenv import get_problem_root, get_supported_problems
+from dmoj.testsuite import Tester
+from dmoj.utils.ansi import ansi_style, print_ansi
+
+all_executors = executors.executors
+
+
+class ProblemTester(Tester):
+    def test_problem(self, problem):
+        self.output(ansi_style('Testing problem #ansi[%s](cyan|bold)...') % problem)
+        fails = 0
+        with open(os.path.join(get_problem_root(problem), 'init.yml'), 'r') as f:
+            config = yaml.safe_load(f.read())
```

Comment:
This is definitely not correct, since it disregards all of the features of our YAML dialect. (This needs to at least be wrapped in a `ConfigNode`, but really this parsing logic seems like it should be bumped into `Problem`.